install-nhc.sh

```diff
@@ -70,12 +70,12 @@ distro_check=$( cat /etc/os-release | grep -i ID_LIKE=)
 distro=`awk -F= '/^NAME/{print $2}' /etc/os-release`
 
 if [[ $distro_check =~ "debian" ]]; then
```

Comment:
This condition targets both Ubuntu 20.0 and 22.04 

I believe the boost install should be the same for both 20.04/22.04 

So, we most likely don't need to differentiate between the two.

Please try to add any additional boost installs to the previous line 73

Reply:
What does this do ```$(apt-cache search libpci-dev | grep -o '^[^ ]*')```?

Reply:
If installing boost will break Alma, then do we still want to install boost only on Ubuntu 20/22?

Reply:
Removed locate as discussed